.eslintrc-platform.json

```diff
@@ -213,7 +213,6 @@
         "react/jsx-no-target-blank": 2,
         "react/jsx-no-undef": 2,
         "react/jsx-pascal-case": 2,
-        "react/jsx-space-before-closing": [2, "never"],
```

Comment:
This rule is deprecated, and already covered by the `react/jsx-tag-spacing` below.